msgcntr/messageforums-hbm/src/java/org/sakaiproject/component/app/messageforums/dao/hibernate/Topic.hbm.xml

```diff
@@ -191,6 +191,9 @@
 	    <property name="restrictPermissionsForGroups">
 	      <column name="RESTRICT_PERMS_FOR_GROUPS" not-null="true" default="0" />
 	    </property>
+      <property name="faqTopic">
```

Comment:
Spacing

Reply:
Spacing vs MSGCNTR, name a better DUO!

Reply:
Yes, I love it when i have to mix tabs with spaces to "fix" the indentation xD